src/client/components/app/Transactions/DepositTx.tsx

```diff
@@ -119,7 +128,7 @@ export const DepositTx: FC<DepositTxProps> = ({
 
   useEffect(() => {
     if (!selectedVault || !selectedSellTokenAddress) return;
-    if (toBN(debouncedAmount).gt(0) && !inputError) {
+    if (toBN(debouncedAmount).gt(0) && !inputError && selectedSellToken) {
```

Comment:
Can we merge these 2 ifs and then just return?

Reply:
Looks better, unrelated to this PR but there are a lot of checks though :)

No action needed